app/assignment/services/assignmentService.js

```diff
@@ -26,6 +26,12 @@ class AssignmentService {
 	
 				return deferred.promise;
 			},
+			updateCourse: function (course) {
+				return ApiService.put("/api/courseView/courses/" + course.id, course);
+			},
+			updateInstructorNote: function (instructorNote) {
+				return ApiService.put("/api/schedules/" + instructorNote.scheduleId + "/instructors/" + instructorNote.instructorId + "/instructorNotes", instructorNote);
```

Comment:
By this API route, are we storing notes per instructor and per schedule?

Reply:
That is correct.